frontend/src/components/GoesData/FetchTab.tsx

```
@@ -31,294 +52,432 @@ function isDateInRange(dateStr: string, avail: SatelliteAvailability): boolean {
-                )}
-              </div>
-            )}
-          </>
-        )}
-      </div>
+            <div className="flex gap-3">
+              <button
+                onClick={() => setShowConfirm(false)}
+                className="flex-1 px-4 py-2 text-sm bg-gray-100 dark:bg-slate-800 text-gray-600 dark:text-slate-300 rounded-lg hover:bg-gray-200 dark:hover:bg-slate-700 transition-colors"
+              >
+                Cancel
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)